code/modules/RCD/schematics/pipe.dm

```diff
@@ -152,8 +152,8 @@
 
 	to_chat(user, "Building gas sensor...")
 	playsound(get_turf(master), 'sound/machines/click.ogg', 50, 1)
-	if(!do_after(user, A, 20))
-		return 1
+//	if(!do_after(user, A, 1))
```

Comment:
No, you should be able to reset the branch and only recommit the changes you need.